tests/data/spacenet/data.py

```diff
@@ -75,7 +75,7 @@ def create_test_image(img_dir: str, imgs: List[str]) -> List[List[float]]:
     """
     for img in imgs:
         imgpath = os.path.join(img_dir, img)
-        Z = np.arange(4, dtype="uint16").reshape(2, 2)
+        Z = np.random.randint(0, 255, size=(256, 256), dtype="uint16")
```

Comment:
What's the minimum size that will work? In most other datasets, we use 32x32 or 64x64.

Reply:
2x2 would work with the epsilon change due to the way we mock the model